finder-app/manual-linux.sh

```diff
@@ -12,6 +12,7 @@ BUSYBOX_VERSION=1_33_1
 FINDER_APP_DIR=$(realpath $(dirname $0))
 ARCH=arm64
 CROSS_COMPILE=aarch64-none-linux-gnu-
+CUR_DIR=$PWD
```

Comment:
@maheshchnt $PWD may not be the directory you want here, it assumes you've called manual-linux.sh from the location where it resides.  You should use something like `$(dirname $0)` instead